Source/WebKit/UIProcess/API/glib/WebKitCookieManager.cpp

```diff
@@ -204,6 +206,7 @@ void webkit_cookie_manager_set_persistent_storage(WebKitCookieManager* manager,
  * @policy: a #WebKitCookieAcceptPolicy
  *
  * Set the cookie acceptance policy of @cookie_manager as @policy.
+ * 
```

Comment:
Another trailing space here:

```suggestion
 *
```